src/components/App.js

```diff
@@ -56,6 +56,7 @@ function App() {
               <Route path="/hackathon" component={Hackathon} />
               <Route path="/hackathon-register" component={HackathonRegister} />
               <Route path="/hackathon-submit" component={HackathonSubmit} />
+              <Route path="/picnic" component={Picnic} />
```

Comment:
can we change the path to bbq or summerbbq (anything works) since we are not promoting it as picnic.